setup.py

```diff
@@ -3,13 +3,13 @@
 
 setup (
     name = "pycog3",
-    version = "0.1.27",
-    scripts = ["bin/cog-command"],
+    version = "0.1.28",
+    scripts = ["bin/cog-command",],
```

Comment:
Unnecessary comma?

Reply:
Cosmetic - this is also an acceptable python syntax.  Removed.